### PR TITLE
OSDOCS#10543: Replace terms with new attributes

### PR DESCRIPTION
**Pull request by xenolinux**
Per https://github.com/openshift/openshift-docs/pull/75783 adding this PR to replace existing terms with their newly created common attributes.

:hcp-capital: Hosted control planes
:hcp: hosted control planes

Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OSDOCS-10543

Link to docs preview:

Welcome page: https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/#hosted-control-plane-activities
Updating_a_cluster: https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-hosted-control-planes.html
Security: https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-management/compliance-operator-installation.html#installing-compliance-operator-hcp_compliance-operator-installation
Configuring ipsec encryption: https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/openshift_network_security/configuring-ipsec-ovn.html
Hosted control planes book:https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/
Architecture book: 
- https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#hosted-control-planes-benefits_control-plane
- https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/architecture/mce-overview-ocp.html#mce-on-ocp

Preparing an Agent-based installed cluster for the MCE:https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.html
Installing the compliance operator on HCP: https://75820--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-management/compliance-operator-installation.html#installing-compliance-operator-hcp_compliance-operator-installation

QE review:
 - Not required

Additional information:

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
LGTM.

@xenolinux Great work on incorporating attributes into the documentation! I can see the effort and attention to detail that went into this. I'd like to share a thought on the PR's scope. Given that the changes affect 30+ files and introduce more than one attribute, this was a lot to traverse and it can be a bit overwhelming for reviewers who are not as familiar with these sections of the documentation as you are. To make the review process smoother and more efficient, it might have been helpful if this work had been broken up into 2-3 different stories with smaller, more manageable PRs. This could help facilitate both the implementation and review processes. Just a suggestion for something to think about next time, but overall, I think the changes are very beneficial. Thanks for all your hard work on this!

**Comment by xenolinux** [comment]
> LGTM.
> 
> @xenolinux Great work on incorporating attributes into the documentation! I can see the effort and attention to detail that went into this. I'd like to share a thought on the PR's scope. Given that the changes affect 30+ files and introduce more than one attribute, this was a lot to traverse and it can be a bit overwhelming for reviewers who are not as familiar with these sections of the documentation as you are. To make the review process smoother and more efficient, it might have been helpful if this work had been broken up into 2-3 different stories with smaller, more manageable PRs. This could help facilitate both the implementation and review processes. Just a suggestion for something to think about next time, but overall, I think the changes are very beneficial. Thanks for all your hard work on this!

@rh-tokeefe I totally agree. I will definitely consider for all of the next times!! Thank you :)

**Review by kcarmichael08 — COMMENTED** [review]
Hi Servesha, it looks like one variable needs to be fixed before merging; otherwise, it looks good! I am going to remove the merge review label and you can re-add it after fixing the issue. If nobody in the other time zones can review it again, I can grab it again tomorrow morning.

File: modules/preparing-an-initial-cluster-deployment-for-mce-connected.adoc [hunk 1]
```diff
@@ -5,9 +5,9 @@
 :_mod-docs-content-type: PROCEDURE
 [id="preparing-an-initial-cluster-deployment-for-mce-connected_{context}"]
 
-= Preparing an Agent-based cluster deployment for the multicluster engine for Kubernetes Operator while connected
+= Preparing an Agent-based cluster deployment for the {mce} while connected
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/preparing-an-initial-cluster-deployment-for-mce-connected.adoc]: Removed {mce} for now.

File: modules/preparing-an-initial-cluster-deployment-for-mce-disconnected.adoc [hunk 2]
```diff
@@ -5,9 +5,9 @@
 :_mod-docs-content-type: PROCEDURE
 [id="preparing-an-initial-cluster-deployment-for-mce-disconnected_{context}"]
 
-= Preparing an Agent-based cluster deployment for the multicluster engine for Kubernetes Operator while disconnected
+= Preparing an Agent-based cluster deployment for the {mce} while disconnected
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/preparing-an-initial-cluster-deployment-for-mce-disconnected.adoc:8]: Removed {mce} for now

**Comment by kcarmichael08** [comment]
everything lgtm now, merging

**Pull request merged**